### PR TITLE
+13 Icons +1 Link

### PR DESCRIPTION
**Pull request by airon-main**
# Description

Second time contributing. there's a change i made on the gradle. i'll change it back if requested.
there are also some apps that i haven't found the activity, also some that i haven't filter which activity works and not. i'll get to them tomorrow, I'm real tired today

### Linked
Calendar (`com.transsion.calendar` -> "calendar") (i can't find the template, so i just guess)

### Added
NeoBank (`com.bnc.finance`)
XArena (`com.transsion.smartpanel`)
Blynk (`cloud.blynk`)
Prayer Times (`com.reworewo.prayertimes`)
Modern Strike Online (`com.gamedevltd.modernstrike`)
Palm Store (`com.transsnet.store`)
Zen Flip Clock (`com.mad.zenflipclock`)
Charge Meter (`dev.km.android.chargemeter`)
Noti Progress Bar (`com.gustavoas.noti`)
Touch the Notch (`com.notch.touch`)
DeskDock (`com.floriandraschbacher.deskdock.free`)
APK Info (`com.wt.apkinfo`)
Roto Force (`com.pid.rotoforce`)

## Contributor's checklist

- [x] I made the icons following [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

File: svgs/noti_progress_bar.svg
Comment (Chefski) [inline svgs/noti_progress_bar.svg:1]: The `n` got converted to a fill shape. Could you change it to a stroke?
image

File: svgs/xarena.svg
Comment (Chefski) [inline svgs/xarena.svg:1]: The icon is a bit bugged on the Github preview. If you can download the debug and check if it's fine in app would be good. The icons are sometimes bugged like on Github
image

**Review by Chefski — CHANGES_REQUESTED** [review]

File: svgs/noti_progress_bar.svg
Comment (airon-main) [reply svgs/noti_progress_bar.svg:1]: yea sure

File: svgs/xarena.svg
Comment (airon-main) [reply svgs/xarena.svg:1]: I have retraced the old vector and still get the same result 😭😭

Comment (airon-main) [reply svgs/xarena.svg:1]: image
i've had enough

**Comment by airon-main** [comment]
I'm just wondering. You know the activities of the app some time have a lot that even the one that has .MainActivity doesn't even work. Is it okay for me to just put all the activity that looks working, or do I need to filter down the non-working ones?

**Comment by airon-main** [comment]
Alr all set. ready to merge

**Comment by airon-main** [comment]
Oh wait. It's not ready, I've deleted the correct activity

**Comment by airon-main** [comment]
Aight, now it's all done

**Comment by airon-main** [comment]
No merge?

**Review by Chefski — APPROVED** [review]

**Pull request merged**